packages/salesforcedx-vscode-core/src/commands/forceGenerateFauxClasses.ts

```diff
@@ -25,16 +26,29 @@ class ForceGenerateFauxClassesExecutor extends SfdxCommandletExecutor<{}> {
   public build(data: {}): Command {
     return new SfdxCommandBuilder()
       .withDescription(nls.localize('force_refresh_sobjects'))
+      .withArg('refresh sObject definitions')
       .build();
   }
 
   public async execute(response: ContinueResponse<{}>): Promise<void> {
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new LocalCommandExecution(this.build(response.data));
+
+    this.attachExecution(execution, cancellationTokenSource, cancellationToken);
+
     const projectPath: string = <string>vscode.workspace.rootPath;
```

Comment:
Use `as string` instead of <string>. http://acdcjunior.github.io/typescript-cast-object-to-other-type-or-instanceof.html. I mentioned this several times already but I'm not sure if you were one of them I mentioned it to. Again, if this is something that if we can enforce at code formatter level, that would be better.

Reply:
will do - I wasn't sure which was the preferred form